src/Agent/NewRelic/Agent/Core/Core.csproj

```diff
@@ -33,21 +33,25 @@
 
   <ItemGroup>
     <PackageReference Include="Autofac" Version="4.5.0" />
-    <PackageReference Include="Google.Protobuf" Version="3.21.12" />
-    <PackageReference Include="Grpc.Tools" Version="2.28.1">
+    <PackageReference Include="Google.Protobuf" Version="3.25.2" />
+    <PackageReference Include="Grpc.Tools" Version="2.60.0">
```

Comment:
This is a very large jump, and I don't know if our integration test coverage is adequate for it.

Reply:
Added infinite tracing tests to the container integration tests to verify protobuf is working across Linux distros.